src/JSONSchema/types/index.ts

```diff
@@ -0,0 +1,4 @@
+export interface JSONSchemaType {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any
```

Comment:
Why this has a type of `any`?

Reply:
I kind of was not bothered to properly create a generic JSON Schema type 😅 as I was starting the project, but I should get right to it.